packages/leva/stories/controlled-inputs.stories.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import { ComponentPropsWithoutRef, forwardRef, useRef } from 'react'
+import { useDrag } from 'react-use-gesture'
```

Comment:
I've cleaned the example a bit by delegating the drag logic to `useDrag` in order to avoid polluting the source code.

Reply:
Thanks!